### PR TITLE
Fix migrations

### PR DESCRIPTION
**Pull request by AlexNisnevich**
There were a few issues getting in the way of being able to run migrations. I remember testing them before, so I'm not sure where things went wrong.

At any rate, the migrations all work on a fresh db after these changes.

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -17,7 +17,7 @@ services:
         - GIT_SHA1
     image: "${DOCKER_REGISTRY_PATH}ursus"
     ports:
-      - "3000:80"
+      - "4000:80"
```
Comment (wetchler) [inline docker-compose.yml]: I assume you meant to revert the changes in this file?

File: config/aws.yml [hunk 2]
```diff
@@ -2,7 +2,7 @@
 development:
   access_key_id: AKIA1111111111111UA
   secret_access_key: secret
-  endpoint: http://localhost:8000
+  endpoint: http://localhost:8010
```
Comment (wetchler) [inline config/aws.yml]: I assume you meant to revert this change?

**Comment by wetchler** [comment]
LGTM modulo the few comments above

File: docker-compose.yml (hunk 1 above)
Comment (AlexNisnevich) [reply docker-compose.yml]: oops

File: config/aws.yml (hunk 2 above)
Comment (AlexNisnevich) [reply config/aws.yml]: oops
